src/super_gradients/examples/predict/detection_predict_with_labels.py

```diff
@@ -0,0 +1,27 @@
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+from pathlib import Path
+from super_gradients.training.datasets import COCODetectionDataset
+
+# Note that currently only YoloX, PPYoloE and YOLO-NAS are supported.
+model = models.get(Models.YOLO_NAS_L, pretrained_weights="coco")
+mini_coco_data_dir = str(Path(__file__).parent.parent.parent.parent.parent / "tests" / "data" / "tinycoco")
+
+dataset = COCODetectionDataset(
+    data_dir=mini_coco_data_dir, subdir="images/val2017", json_file="instances_val2017.json", input_dim=None, transforms=[], cache_annotations=False
+)
+
+# x's are np.ndarrays images of shape (H,W,3)
```

Comment:
drop the comments please

Reply:
updated in de413502d01b617481b509f27da06fe092ec1fa1